_data/team.yml

```diff
@@ -234,6 +240,12 @@
   type: community
   fingerprint: 
 
+- name: Simon Newton
+  email: simon.newton@gmail.com
```

Comment:
@mfc  Can we use my email theplexus@ctemplar.com please? sorry to be a pain ;)

Reply:
yep done, apologies for using wrong address!